.pre-commit-config.yaml

```diff
@@ -1,25 +1,22 @@
 default_language_version:
```

Comment:
can you help me understand what are we are trying to achieve with the changes in this file please?

Reply:
It's a working trick to make pre-commit work for both 3.7 and 3.8.
As my M1 is not supporting 3.7 if I run `pre-commit`, or when the hook runs I have this error:
```
 RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.7'
```
Declaring `default_language_version: python3` will work for `python 3.7` (that non-M1 dev have) and above versions for M1 users. 
Let me know if you want me to open a separate PR for this. I committed this as it's mandatory to use the git hook.